mail-filter/bogofilter/bogofilter-1.2.5.ebuild

```diff
@@ -0,0 +1,129 @@
+# Copyright 1999-2021 Gentoo Authors
+# Distributed under the terms of the GNU General Public License v2
+
+EAPI=7
+
+inherit autotools db-use flag-o-matic toolchain-funcs
+
+DESCRIPTION="Bayesian spam filter designed with fast algorithms, and tuned for speed"
+HOMEPAGE="http://bogofilter.sourceforge.net/"
+SRC_URI="mirror://sourceforge/${PN}/${P}.tar.xz"
+
+LICENSE="GPL-3+ CC-BY-ND-3.0"
+SLOT="0"
+KEYWORDS="~alpha ~amd64 ~arm ~arm64 ~hppa ~ia64 ~ppc ~ppc64 ~sparc ~x86"
+IUSE="berkdb sqlite tokyocabinet"
+
+# pax needed for bf_tar
+DEPEND="
+	app-arch/pax
+	sci-libs/gsl:=
+	virtual/libiconv
+	berkdb?  ( >=sys-libs/db-3.2:* )
+	!berkdb? (
+		sqlite?  ( >=dev-db/sqlite-3.6.22 )
+		!sqlite? (
+			tokyocabinet? ( dev-db/tokyocabinet )
+			!tokyocabinet? ( >=sys-libs/db-3.2:* )
+		)
+	)
```

Comment:
As much as I hate `REQUIRED_USE`, maybe one could be used here. Well for me the `elogs` are enough. Thanks!